src/docs/asciidoc/http-client.adoc

```diff
@@ -429,6 +429,40 @@ public class Application {
 ----
 ====
 
+
+== Boostrap Configuration 
+
+Use `HttpClient.from` and create a `TcpClient` when you need more control of your client configuration, i.e., when you 
+want to specify the local address.
+
+
+====
+[source,java]
+----
+import reactor.netty.http.client.HttpClient;
+import reactor.netty.http.client.HttpClientResponse;
```

Comment:
import reactor.netty.http.client.HttpClient;
import reactor.netty.tcp.TcpClient;

import java.net.InetSocketAddress;

Reply:
let's have the proper imports